.travis.yml

```diff
@@ -28,34 +26,26 @@ jobs:
         - cmake . -DCMAKE_C_FLAGS="$COMPILER_FLAGS" -DCMAKE_CXX_FLAGS="$COMPILER_FLAGS"
         - make -j2
         - ./check.py --test-waterfall
-      env: |
-        CC_COMPILER="./test/wasm-install/wasm-install/bin/clang"
-        CXX_COMPILER="./test/wasm-install/wasm-install/bin/clang++"
 
     - <<: *test-ubuntu
       env: |
-        CC_COMPILER="./test/wasm-install/wasm-install/bin/clang"
-        CXX_COMPILER="./test/wasm-install/wasm-install/bin/clang++"
         COMPILER_FLAGS="-fsanitize=undefined -fno-sanitize-recover=all -fsanitize-blacklist=$(pwd)/ubsan.blacklist"
 
+    # FIXME we currently must disable leak detection, see #1351
     - <<: *test-ubuntu
       env: |
-        CC_COMPILER="./test/wasm-install/wasm-install/bin/clang"
-        CXX_COMPILER="./test/wasm-install/wasm-install/bin/clang++"
-        COMPILER_FLAGS="-fsanitize=address"
+        COMPILER_FLAGS="-fsanitize=address" ASAN_OPTIONS="detect_leaks=0"
```

Comment:
Do you know how this interacts with line 18  above? I guess this must override that so maybe we should add symbolize=1 here too.

Reply:
Good catch. I can just move this to there I guess, seems safer for future changes.

Reply:
But nevermind, this didn't fix it anyhow. LSAN is just crashing entirely, not in a way affected by disabling the leak check feature. Pushed a commit to do it the original way again.